packages/puppeteer-schematics/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "puppeteer-schematics",
```

Comment:
WDYT you think if we name the package `@pptr/ng-schematic` and the folder name can be `ng-schematic`